pai-management/src/yarn-exporter/dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+FROM ubuntu:16.04
+
+#
+# Preparation
+#
+
+WORKDIR /root/
+
+RUN apt-get update && \
+    apt-get -y install wget build-essential python3 python3-pip git pciutils
+
+COPY copied_file/yarn_exporter.py /usr/local/
+COPY ./requirements.txt /usr/local/
+COPY ./run.sh /usr/local/
+
+RUN pip3 install -r /usr/local/requirements.txt
+
+#
+# start
+#
+
+ENTRYPOINT ["/usr/local/run.sh"]
```

Comment:
I think it's better to specify command & args in [configuration file](https://kubernetes-v1-4.github.io/docs/user-guide/configuring-containers/#launching-a-container-using-a-configuration-file)

Reply:
Resolved